src/python/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod pyproject;
```

Comment:
Can this be moved into `providers/python`. Ideally there are no language specific modules outside of the providers directory. The core nixpacks logic and directory structure should be as language agnostic as possible.